### DISCUSSION TITLE
Getting 2 IDs  with MongoDB; Binary and  ObjectID

### DISCUSSION THREAD

**Discussion by migoya2020** [fastapi-users/fastapi-users#569] [Q&A]
Why am i getting this?
 `{
    "_id": {
        "$oid": "605bb2de31da4a0306318ec8"
    },
    "id": {
        "$binary": {
            "base64": "qIIjhYOvTU+UI/TE+2ml4A==",
            "subType": "04"
        }
    },
    "email": "king.arthur@camelot.bt",
    "is_active": true,
    "is_superuser": false,
    "is_verified": false,
    "hashed_password": "$2b$12$W4hskDIy4oDs.41TpIIJf.yc3g8f54QieOj8GiEZAwPejZERlEHDC"
}`

**Comment by frankie567** [comment]
Could you provide more details about the context in which you get this?
